<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Menu</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="btnGapoktan">
      <property name="text">
       <string>Gapoktan</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnPoktan">
      <property name="text">
       <string>Poktan</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnPetani">
      <property name="text">
       <string>Petani</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnPenyuluh">
      <property name="text">
       <string>Penyuluh</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnLokasi">
      <property name="text">
       <string>Lokasi</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
